### PR TITLE
Login

### PR DESCRIPTION
**Pull request by r-nsubuga**
What does this PR do?
A user can login into the system

Description of Task to be completed?
Enter the username and password to access the system

How should this be manually tested?
Enter incorrect information to check validity and authentication

Screenshots (if appropriate)
Screenshot from 2023-07-17 15-35-51

### CODE REVIEW

File: src/main/webapp/resources/images/kimwanyi.jpg
Comment (KSConrad) [inline src/main/webapp/resources/images/kimwanyi.jpg:1]: They would love this Logo

File: src/main/webapp/pages/authentications/LoginForm.xhtml
Comment (KSConrad) [inline src/main/webapp/pages/authentications/LoginForm.xhtml:1]: Screenshot 2023-07-17 at 7 33 00 PM

You can use that snippet to help you position your form to the centre of the screen

**Review by KSConrad — APPROVED** [review]
You should consider adding more fields... Maybe. It is possible you have flow for capturing other information about a member, is it?

**Comment by r-nsubuga** [comment]
i also made the register form which can capture more details about the user

**Pull request merged**